pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: tiled
-version: 0.2.1
+version: 0.2.2
```

Comment:
since this is a breaking change, we should do a bigger release (like 0.3). please revert this change as we can bump the version when we are ready to release all the changes

Reply:
but please add your change to the CHANGELOG.md file under `<next>`

Reply:
ok, done this.